src/contracts/core/DelegationManager.sol

```diff
@@ -510,15 +534,23 @@ contract DelegationManager is
         delegatedTo[staker] = operator;
         emit StakerDelegated(staker, operator);
 
-        (IStrategy[] memory strategies, uint256[] memory shares) = getDelegatableShares(staker);
+        (IStrategy[] memory strategies, uint256[] memory shares) = getDelegatableScaledShares(staker);
```

Comment:
call this `scaledShares`

Reply:
But they're not scaled at this point, because the staker isn't delegated? If i understand the comment below correctly

Calling the "getXScaled" function here is confusing tho...

Reply:
They are technically "scaled" but because its scaled based on operator address(0) there is no difference
because the following is true `scaleShares(shares) == shares`.

So rather than doing `descaledShares = descaleShares(scaledShares)` prior to scaling shares according to the new delegated operator, we can just skip descaling the shares.

Reply:
I used the variable name here as `shares` to indicate it is in a sense already "descaled" but I admit it is very confusing

Reply:
ah yes, these aren't scaled. confusing definitely